src/sentry/integrations/aws_lambda/utils.py

```diff
@@ -86,18 +131,16 @@ def get_function_layer_arns(function):
 
 def get_latest_layer_for_function(function):
     region = parse_arn(function["FunctionArn"])["region"]
-    runtime = function["Runtime"]
-    if runtime.startswith("nodejs"):
```

Comment:
I feel like you could have kept this part here but instead have `if not runtime.startswith("nodejs")` and then raise

Reply:
@MeredithAnya No need, `get_option_value` will still end up raising the same error (the check was just moved).

Reply:
@scefali yeah I guess just felt like there was already a lot going on in `get_option_value` and it looked like you didn't need the option value if that was the case (not node)

Reply:
@MeredithAnya Understood. Eventually that function is going to look something like this:

```
    if runtime.startswith("nodejs"):
        prefix = "node"
    elif runtime in ["python3.8", "python3.7", "python3.6", "python2.7"]:
       prefix = runtime
    else:
        raise Exception("Unsupported runtime")
```
This is because Python has a different layer for each runtime while Node does not. So we really can't avoid mapping the runtime to the option/cache prefix inside of  `get_option_value` which makes it the logical place to raise the error.

Reply:
yeah okays makes sense maybe eventually it could be pulled out into its own check like 

```python
def get_runtime_prefix(runtime):
    # get prefix or raise
    return prefix
```

and then you could do something like 

```python
prefix = get_runtime_prefix(function["Runtime"])
get_option_value(function, prefix, option)
```

that's where my head was at but didn't see that you also use it in `get_latest_layer_version`. anyhoo, all stuff to think about later when supporting more than just node